cypress/fixtures/contracts/contractsDashboard.json

```diff
@@ -171,7 +171,7 @@
   },
   {
     "contractReference": "127-127-1277",
-    "terminationDate": "2025-09-15T23:00:00Z",
+    "terminationDate": "2025-09-17T23:00:00Z",
```

Comment:
This test will fail again in a day or two, but this is a quick-fix to get this feature through - this test will require some more time and care to fix carefully